main/res/layout-w1200dp/coordinatescalculate_dialog.xml

```diff
@@ -79,7 +79,7 @@
                     android:overScrollMode="always"
                     android:scrollbarStyle="insideInset"
                     android:scrollbars="vertical"
-                    android:inputType="textAutoCorrect"
+                    android:inputType="textMultiLine"
```

Comment:
Did you see any advantage with textAutoCorrect compared with plain text?
You can combine the value textMultiLine with other text entries, e.g.
"textMultiLine|textAutoCorrect"
But yy suggestion here is "textMultiLine|textCapSentences", because the text from the coordinate calculator is copied in the user_note field of the editwaypoint_activity, which is defined in this way.

Reply:
Thanks for the hint. Will merge here first and append your suggestion next week when back at PC. Or feel free to add a PR yourself.